ts/src/bitget.ts

```diff
@@ -2227,6 +2230,10 @@ export default class bitget extends Exchange {
          * @method
          * @name bitget#fetchTrades
          * @description get the list of most recent trades for a particular symbol
+         * @see https://bitgetlimited.github.io/apidoc/en/spot/#get-market-trades
```

Comment:
@Dan-krm we have these parameters swapped here 🤦 don't know how he didn't notice before
![image](https://github.com/ccxt/ccxt/assets/43336371/24965a29-f769-488b-934c-dd7a959be047)